RPGManager_development/test/unit_test/character_sheet_unit_tests/ArmorsUnitTests.java

```diff
@@ -0,0 +1,145 @@
+		thrown.expect(exceptionClass);
+		thrown.expectMessage(message);
+	}
+
+	// default Armors for BaseClasses
+
+	@Test
+	public void testGetDefaultArmorForWarrior() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.WARRIOR);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.HEAVY_LEATHER, actualValue);
+	}
+
+	@Test
+	public void testGetDefaultArmorForRogue() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.ROGUE);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.LIGHT_LEATHER, actualValue);
+	}
+
+	@Test
+	public void testGetDefaultArmorForMage() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.MAGE);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.ROBE, actualValue);
+	}
+
+	// default ArmorRating for BaseClasses
+
+	@Test
+	public void testGetDefaultArmorRating() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(4, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	@Test
+	public void testGetDefaultArmorRatingWarrior() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.WARRIOR);
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(4, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	@Test
+	public void testGetDefaultArmorRatingRogue() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.ROGUE);
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(3, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	@Test
+	public void testGetDefaultArmorRatingMage() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		characterSheet.setData(Fields.BASECLASS, BaseClasses.MAGE);
+		int actualArmorValue = characterSheet.getData(Fields.ARMOR_RATING);
+		assertEquals(0, actualArmorValue + characterSheet.<Armors>getData(Fields.ARMOR_TYPE).getArmorRating());
+	}
+
+	// Functional tests
+
+	@Test
+	public void testSetArmorTypeLightPlate() {
+		CharacterSheet characterSheet = new CharacterSheet("characterSheet");
+		Armors expectedArmor = Armors.LIGHT_PLATE;
+		characterSheet.setData(Fields.ARMOR_TYPE, expectedArmor);
+		Armors actualValue = characterSheet.getData(Fields.ARMOR_TYPE);
+		assertEquals(Armors.LIGHT_PLATE, actualValue);
+	}
+
+	@Test

```

Comment:
Itt az, hogy mely BaseClass számára állítasz be ARMOR_TYPE-ot az lényegtelen az ARMOR_TYPE settelése szempontjából nem? Erre a 3 tesztre (testSetArmorForWarrior, testSetArmorForRugue, testSetArmorForMage) miért van szükség?